app/main.py

```
@@ -20,13 +20,58 @@
+            body_str = body.decode("utf-8")
+            if len(body_str) <= MAX_LOG_BODY_SIZE:
+                body_json = json.loads(body_str)
+                if contains_sensitive_keys(body_json):
+                    return "[REDACTED: contains sensitive data]"
+                return body_json
+            return f"[TRUNCATED: {len(body_str)} bytes]"
+        except (json.JSONDecodeError, UnicodeDecodeError):
+            if len(body) <= MAX_LOG_BODY_SIZE:
+                return body.decode("utf-8", errors="replace")
+            return f"[BINARY DATA: {len(body)} bytes]"
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)